CHANGES.md

```diff
@@ -1,3 +1,149 @@
+## stable-2.2.0
+
+This stable release introduces several major improvements, including support for
+timeouts and retries via service profiles, more flexible targeting for automatic
+proxy injection, and new CLI subcommands, such as `logs` and `endpoints`, that
+provide better visibility into the Linkerd control plane. It also adds support
+for some experimental features, including a Linkerd CNI plugin that subsumes the
+responsibility of proxy init containers in the data plane.
```

Comment:
Funny, we've written a very similar paragraph for the announcement blog post. They don't have to be the same, but here it is for reference. (I added the `logs` and `endpoints` blurb after seeing your version. I think the biggest difference now is the mention of the client identity header.)

> This major release introduces automatic request retries and timeouts, and graduates auto-inject to be a fully-supported (non-experimental) feature. It adds several new CLI commands, including logs and endpoints, that provide diagnostic visibility into Linkerd's control plane. Finally, it introduces two exciting experimental features: a cryptographically-secured client identity header, and a CNI plugin that avoids the need for NET_ADMIN kernel capabilities at deploy time. Of course, this release also includes a ton of bugfixes and improvements across the board.

Reply:
Heh, I was trying my best to sounds like @wmorgan. Now I can just use your words directly :)